rules/import-path-order.js

```diff
@@ -0,0 +1,236 @@
+'use strict';
+
+const coreModules = require('resolve/lib/core');
```

Comment:
I have a better list for this here: https://github.com/sindresorhus/builtin-modules and https://github.com/sindresorhus/is-builtin-module

Can you also add `electron` as a built-in so it's sorted before third-party packages?

Reply:
Ha, why am I not surprised you have a package for this. 🦄